fawltydeps/settings.py

```diff
@@ -242,8 +242,13 @@ def print_toml_config(settings: Settings, out: TextIO = sys.stdout) -> None:
         "# Copy this TOML section into your pyproject.toml to configure FawltyDeps",
         "# (default values are commented)",
         "[tool.fawltydeps]",
-    ] + [
-        f"{'# ' if has_default_value[name] else ''}{name} = {value!r}"
-        for name, value in simple_settings.items()
     ]
+    for name, value in simple_settings.items():
+        if value is None:
+            continue
+        line = f"{'# ' if has_default_value[name] else ''}"
+        if isinstance(value, bool):
+            lines.append(line + f"{name} = {str(value).lower()}")
+        else:
+            lines.append(line + f"{name} = {value!r}")
```

Comment:
After a second look, I wonder if this is closer to what we want:

```suggestion
    toml_value_map = {
        None: "...",  # always commented, see sanity check below
        False: "false",
        True: "true",
        # add more values here for which repr(value) is not valid TOML
    }
    for name, value in simple_settings.items():
        prefix = "# " if has_default_value[name] else ""
        if value is None:
            # sanity check: None values are represented in TOML by omission, 
            # hence make sure these are always commented (i.e. equal to default)
            assert has_default_value[name]
        toml_value = toml_value_map.get(value, repr(value))
        lines.append(f"{prefix}{name} = {toml_value}")
```

Reply:
Adopted your suggestion for using a `toml_value_map` dict with some needed changes, and rebasing on main (in which this code has been changed). See revised [cd44db9](https://github.com/tweag/FawltyDeps/pull/300/commits/cd44db9ed259b358338a61341975c8990a2df9f6).